09-confidence-intervals.Rmd

```diff
@@ -502,7 +502,7 @@ You can think of a confidence interval as playing the role of a net when fishing
 
 Point estimate/sample statistic           |  Confidence interval
 :-------------------------:|:-------------------------:
-![](images/spear.jpg){ height=1.7in }  |  ![](images/net2.jpg){ height=1.7in }
+![](images/ss/350268308.png){ height=1.7in }  |  ![](images/ss/176684936.png){ height=1.7in }
```

Comment:
is there a photo where the it is more obvious that there is a spear? if that's too restrictive a search criteria, maybe not have both photos involve fishing, but just spear and net?

Reply:
spear:  149730074